hack/verify-upstream-commits.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+OS_ROOT=$(dirname "${BASH_SOURCE}")/..
+source "${OS_ROOT}/hack/common.sh"
+
+"${OS_ROOT}/hack/build-go.sh" cmd/rebasehelpers/commitchecker
+
+# Find binary
+commitchecker=$( (ls -t _output/local/bin/$(os::build::host_platform)/commitchecker) 2>/dev/null || true | head -1 )
```

Comment:
yuck, but we're going to want to do it for all of the helpers we build this way.  make a bash function?